### ISSUE TITLE
Feature request: making output more readable

### ISSUE THREAD

**Issue by Banaanhangwagen** [rtravis/SkypeCacheViewer#12]
For reasons of readability/sorting/filtering, wouldn't it be nice to make the output "CSV-ready"?
On one line you'll have all the details and message. 
One can simply redirect the output to a csv document then.

Example of output:
(do not mind the changes of `skypeTimestampToString`)
![image](https://user-images.githubusercontent.com/25983612/118483454-50576380-b716-11eb-96ae-7d10b4e74402.png)

This is what I changed [in your code](https://github.com/rtravis/SkypeCacheViewer/blob/master/src/skype_leveldb_scanner.cpp#L524):
```
(...)
		if (k == "messagetype") {
			auto const &mtype = std::get<std::string>(val.vt_);
			messageOk = (mtype == "RichText" || mtype == "Text");
		} else if (k == "cuid" || k == "conversationId" || k == "creator") {
			os << std::get<std::string>(val.vt_) << ';';
		} else if (k == "createdTime" || k == "composeTime") {
			os << skypeTimestampToString(std::get<uint64_t>(val.vt_)) << ';';
		} else if (k == "content") {
			os << std::get<std::string>(val.vt_);
		}
	}
```

What do you think ?

**Comment by rtravis** [comment]
Thank you for your feedback, I think CSV output for messages is a good idea and I will integrate your changes this weekend. I might use a different field separator though, (tab?) because the semicolon (';') might appear in some messages.

**Comment by rtravis** [comment]
Now you can use the "-csv" argument to the program to display the messages in CSV format. This change is available in commit fa627ab0a52.
